src/ast/TAGS.ts

```diff
@@ -1,13 +1,22 @@
 import NODE from "./NODE";
+import TAG from "./TAG";
 
 export default class TAGS extends NODE {
-  parse() {
-    // stub
-    throw new Error("Not implemented");
-  }
+  tags: TAG[] = [];
 
-  evaluate() {
-    // stub
-    throw new Error("Not implemented");
+  parse() {
+    this.tokenizer.getAndCheckToken("add Tags");
+    this.tokenizer.getAndCheckToken(":");
+    while (
+      this.tokenizer.moreTokens() &&
+      !this.tokenizer.checkToken("add Color") &&
+      !this.tokenizer.checkToken("add Direction") &&
+      !this.tokenizer.checkToken("add Alignment") &&
```

Comment:
I feel like we can make "add" a keyword, so it would be it's own token and you can just check this.tokenizer.checkToken("add")

Reply:
Hi! the reason I didn't use "add" as a keyword is because I'm going to have to check the next word and see if it's "Color", "Tag", "Direction", or "Alignment" anyways. Splitting out "add" actually adds an extra step